source/style-guide/style/version.txt

```diff
@@ -0,0 +1,140 @@
+.. _version-numbers:
+
+==================
```

Comment:
[nit]

Can we adjust the `=`'s here so they match the title length?

Reply:
Fixed